hieradata/common.yaml

```diff
@@ -22,7 +22,7 @@ deployable_applications: &deployable_applications
   - courts-api
   - courts-frontend
   - designprinciples
-  - EFG
+  - efg-training
```

Comment:
This is intended. EFG will have a separate deploy job which triggers the deploy app job (upon SUCCESS) with EFG_TRAINING as the `TARGET_APP`